### DISCUSSION TITLE
Direct Contact to github user TTECK

### DISCUSSION THREAD

**Discussion by bergquelle** [community/community#124756] [New to GitHub]
Hello, as a newby I've not found an easy way to contact TTECK - I appriciate any help. Thanks - Volker

**Comment by davevad93** [comment]
Hi @bergquelle ,

Do you mean contacting GitHub support? If so, you can open a ticket [here](https://support.github.com/) or contact them directly by email: [support@github.com](mailto:support@github.com)

**Comment by bergquelle** [comment]
Hello @davevad93,

Thanks for your reply. Unfortunately I have no success to contact github support. 
With this link it is like riding a carousel. (https://support.github.com/)
I'll never reach a place, where I can type in my problem / question.

And sending a mail to: support@github.com returns always: 
The answer from the mailer-deamon: Message bounced due to organizational settings.


HERE is what I tried to send and maybe you could be so kind and help me on that question.

Hello Support Team,

I installed Proxmox on a standalone Notebook and I used the script from a github user TTECK (https://github.com/tteck/Proxmox) for installing paperless-ngx. Everything works fine so far.
Now I need to modify certain script files and I do not know how to access them via proxmox. 
Using a shell in Proxmox does not help me at all, because I cannot find the place where paperless-ngx is located. BTW - I am not an expert in UNIX.
That's why I am trying to contact TTECK - BUT HOW???

I've been searching the internet for this question - but no success - I hope you can help me.

Kind Regards,

Volker


Thanks again for your help in advance.

Greetings Volker

**Reply by davevad93** [reply]
Hey @bergquelle I see, I don't know how you might contact directly with TTECK, the only solution I can think of is opening an `issue` on his [repo](https://github.com/tteck/Proxmox/issues) explaining your problem, hope it helps.

**Comment by bergquelle** [comment]
Hi davevad93, I'll give it a try - I'll let you know.

Thanks

**Comment by bergquelle** [comment]
Hi davevad93, the hint with the repo helped - I could find someone to give me the right answers.
Thanks for your help too.

**Reply by davevad93** [reply]
Hey @bergquelle , you're welcome I'm glad to have been helpful. Have a nice day!
